lesson_01/hello.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/kyokomi/emoji/v2"
+)
+
+func main() {
+	emojiKeyword := flag.String("e", "Hello, World :smile:\n", "")
```

Comment:
Why is this so hard?=)
We just need to print some symbols to terminal, not to have an opportunity to print anything.
Just imao XD